src/components/FabAccountButtonBar.tsx

```diff
@@ -84,7 +85,13 @@ function FabAccountButtonBar({
       const { navigationParams, confirmModalProps } = data;
       if (!confirmModalProps) {
         setInfoModalProps();
-        if (navigationParams) onNavigate(...navigationParams);
+        if (navigationParams) {
+          if (typeof navigationParams === "string") {
+            Linking.openURL(navigationParams);
```

Comment:
better use another field like deelink instead of having mixed type fields :pray: